draft-ietf-quic-tls.md

```diff
@@ -501,9 +501,11 @@ handshake, new data is requested from TLS after providing received data.
 As keys for new encryption levels become available, TLS provides QUIC with those
 keys.  Separately, as keys at a given encryption level become available to TLS,
 TLS indicates to QUIC that reading or writing keys at that encryption level are
-available.  These events are not asynchronous; they always occur immediately
-after TLS is provided with new handshake bytes, or after TLS produces handshake
-bytes.
+available.  While waiting for TLS to provide these keys, an endpoint SHOULD
+buffer the received packets marked as protected by the keys yet to be provided,
+and process them once those keys become available.  If the keys are generated
+asynchronously, an endpoint MAY continue responding to the received packets that
+are processable while waiting for TLS to provide these keys.
```

Comment:
I think that this fixates on the wrong aspect of the problem.  This text can instead just say "if keys are not yet available, and endpoint MAY continue to process those packets for which it has keys".  Or similar.

Reply:
Thanks, I agree that that's better. Applied in 16038c1.